package.json

```diff
@@ -8,7 +8,7 @@
     "esbuild": "^0.25.5",
     "govuk-frontend": "^5.10.2",
     "idb": "^8.0.3",
-    "nhsuk-frontend": "^9.6.2",
+    "nhsuk-frontend": "9.6.1",
```

Comment:
Should we just pin it at this version? v10 is imminent and will require a PR/piece of work to upgrade and update a few things, so we can unpin this dependency then.